specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/examples/operations.json

```diff
@@ -4,7 +4,17 @@
     "subscriptionId": "EA19050C-51FC-4B55-84E5-9EA7283601DA",
     "resourceGroupName": "TestResourceGroup",
     "endpoint": "{endpoint}",
-    "Ocp-Apim-Subscription-Key": "{API key}"
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "$top": 20,
+    "$skip": 10,
+    "$maxpagesize": 10,
+    "$orderBy": "CreatedDateTime asc",
+    "createdDateTimeUtcStart": "2021-03-23T00:03:38.0136316",
+    "createdDateTimeUtcEnd": "2021-03-25T00:03:38.0136316",
```

Comment:
test needs to be updated to match the `id` and the created time window

Reply:
this actually makes me wonder, what is the priority the service is using to evaluate the parameters.
Do all parameters need to be met in order to get a result? (in this case, `id`, `createdDateTime` window, and `status` need to match?)

Reply:
yes they are AND parameters

Reply:
ok, i'll update the example so it actually make sense.